### DISCUSSION TITLE
Learning 4 CasADi Framework

### DISCUSSION THREAD

**Discussion by nacereddinemoumai** [casadi/casadi#3814] [General]

**Comment by tmmsartor** [comment]
Hello @nacereddinemoumai, not sure if it is intentional but your post is empty at the moment, have you already solved your problem?

**Reply by nacereddinemoumai** [reply]
Hello @tmmsartor, Thank you for your question. I have already asked my question [here](https://github.com/casadi/casadi/discussions/3275) My current question is how to add a dimension to my input matrix using CasADi?
